environment_hpc.yml

```diff
@@ -2,30 +2,34 @@ name: babs
 channels:
   - conda-forge
 dependencies:
-  - apptainer
+  - python=3.11
   - git
   - git-annex
-  - pip
+  - apptainer
```

Comment:
We've found that the conda-forge apptainer doesn't work reliably on our HPC. I imagine that this would be true for other HPCs as well, since the admins set up the site's auto-mounts and other cluster-specific things